crates/flight_datafusion/src/lib.rs

```diff
@@ -0,0 +1,330 @@
+#![allow(clippy::missing_errors_doc)]
```

Comment:
This file still has duplication with sql_provider_datafusion.

The idea is to wait until MItch's branch merged, then work on another refactoring. As, for flight client, there is no need for the concept the ConnectionPool, also it doesn't need to have the implement DbManager in SQLTable too.

Right now, please ignore the duplication.

Reply:
As mentioned in another comment - we will eventually need the connection pool for FlightClient

Reply:
yeah will do in the refactoring